osf-cas/templates/configmap.yaml

```diff
@@ -319,6 +319,352 @@ httpd/conf.d/shib.conf: |-
   </VirtualHost>
 {{- end -}}
 {{- define "cas.jetty.inlineconfigs" }}
+config/cas.properties: |-
```

Comment:
The only difference between `newCAS` and `oldCAS` is that the file is under `etc/cas/config` instead of `etc/cas`. This file is a copy of [`cas.properties`](https://github.com/CenterForOpenScience/osf-cas/blob/feature/basic-login/etc/cas/config/cas.properties).